cmd/cli/plugin/cluster/credentials_update.go

```diff
@@ -31,10 +35,18 @@ var credentialsUpdateCmd = &cobra.Command{
 	SilenceUsage: true,
 }
 
+const (
+	vsphereProvider = "vsphere"
+	azureProvider   = "azure"
+)
+
 func init() {
 	credentialsUpdateCmd.Flags().StringVarP(&updateCredentialsOpts.namespace, "namespace", "n", "", "The namespace of the cluster to be updated")
 	credentialsUpdateCmd.Flags().StringVarP(&updateCredentialsOpts.vSphereUser, "vsphere-user", "", "", "Username for vSphere provider")
 	credentialsUpdateCmd.Flags().StringVarP(&updateCredentialsOpts.vSpherePassword, "vsphere-password", "", "", "Password for vSphere provider")
+	credentialsUpdateCmd.Flags().StringVarP(&updateCredentialsOpts.azureTenantID, "azure-tenant-id", "", "", "ID for Azure Active Directory in which the app for Tanzu Kubernetes Grid is created")
```

Comment:
Does these help information is same as the --ui help information? If not, please align with --ui help information.